services/api-server/src/simcore_service_api_server/services_http/webserver.py

```diff
@@ -185,14 +186,16 @@ async def _page_projects(
         limit: int,
         offset: int,
         show_hidden: bool,
-        search: str | None = None,
+        search_by_project_name: str | None = None,
     ) -> Page[ProjectGet]:
         assert 1 <= limit <= MAXIMUM_NUMBER_OF_ITEMS_PER_PAGE  # nosec
         assert offset >= 0  # nosec
 
         optional: dict[str, Any] = {}
-        if search is not None:
-            optional["search"] = search
+        if search_by_project_name is not None:
```

Comment:
@bisgaard-itis we need to implement the table that relates projects with jobs asap and get rid of this search! I cannot find any issue. Do you remember where we added this?